.github/workflows/pr_qc.yml

```diff
@@ -24,7 +24,6 @@ jobs:
       - run: |
           npm ci;
           npm run qa;
-          npm run build;
```

Comment:
We should add this back.  The point of running it as part of QC is to verify that the changes in the PR can/do build.

Reply:
Yes! It got left undone after I bumped the VM size. Adding it back!